server/store/flow_store.go

```diff
@@ -3,9 +3,10 @@ package store
 import "fmt"
 
 const (
-	UpdateStatusPropertyName    = "update_status"
-	GetConfirmationPropertyName = "get_confirmation"
-	SubscribePropertyName       = "subscribe"
+	UpdateStatusPropertyName               = "update_status"
+	GetConfirmationPropertyName            = "get_confirmation"
+	ReceiveNotificationsWhileOnMeetingName = "receive_notifications"
```

Comment:
Nit: By `type`ing these it would be more clear that these form a group.

Reply:
Not sure what you mean with this. Do you mind to elaborate?

Reply:
I'm think he means something like

```go
type SettingProperty string

const (
	UpdateStatusPropertyName SettingProperty = "update_status"
```

Reply:
Then anywhere else in the code, you can reference these values as `SettingProperty` instead of `string`

Reply:
Exactly that @mickmister. Thanks for making this clear.

Reply:
Making this change now would imply touching several files (6 + mocks), and feel a bit out of this PR scope. I will take note of this improvement, and apply it to the extracted flow and settings.